blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -38,16 +46,18 @@ export class RunDetailsPipeline extends Component {
 
         this.mergedConfig = this.generateConfig(this.props);
 
-        // It should really be using capability using /rest/classes API
-        const supportsNode = result && result._class === 'io.jenkins.blueocean.rest.impl.pipeline.PipelineRunImpl';
-        if (supportsNode) {
-            fetchNodes(this.mergedConfig);
-        } else {
-            // console.log('fetch the log directly')
-            const logGeneral = calculateRunLogURLObject(this.mergedConfig);
-            // fetchAll indicates whether we want all logs
-            const fetchAll = this.mergedConfig.fetchAll;
-            fetchLog({ ...logGeneral, fetchAll });
+        if (result.state !== 'QUEUED') {
```

Comment:
seeing this everywhere feels bad, but I understand why... 

QUEUED Is bad and Jenkins should feel bad. Could always have a function oneDayShouldNotNeedToKnowIfQueued(result).... ;)